src/main/java/core/basesyntax/RectangleSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class RectangleSupplier {

```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line above the class declaration.